hoot-core/src/main/cpp/hoot/core/io/BulkInsert.cpp

```diff
@@ -22,6 +22,6 @@
  * This will properly maintain the copyright information. DigitalGlobe
  * copyrights will be updated automatically.
  *
- * @copyright Copyright (C) 2015 DigitalGlobe (http://www.digitalglobe.com/)
+ * @copyright Copyright (C) 2016 DigitalGlobe (http://www.digitalglobe.com/)
```

Comment:
That looks good to me Dave.  I think that just because we may have lost some history in the git history doesn't mean we should delete the years in the copyright notice.  I look forward to seeing that change.